vulture_os/services/frontend/models.py

```diff
@@ -2449,7 +2455,10 @@ def generate_server_conf(self):
         """ Generate directive configuration of "server" HAProxy
          :return     String - HAProxy server configuration parameter
          """
-        return "{}:{}".format(JAIL_ADDRESSES['rsyslog'][self.network_address.family], self.rsyslog_port)
+        result = f"{JAIL_ADDRESSES['rsyslog'][self.network_address.family]}:{self.rsyslog_port}"
+        if self.frontend.has_haproxy_conf and self.frontend.healthcheck_service:
```

Comment:
No need to check if the Listener has an Haproxy conf: this method is called to generate one
```suggestion
        if self.frontend.healthcheck_service:
```

Reply:
Suggestion applied, thanks !